### PR TITLE
Adds support for custom identity resources and claims

### PR DESCRIPTION
**Pull request by vitalyal**
Hi, 
Here is a PR with support for custom identity resources and claims. Config could look like:
`USERS_CONFIGURATION_INLINE: |
        [
          {
            "SubjectId":"1",
            "Username":"User1",
            "Password":"pwd",
            "Claims":[{
              "Type": "tenantId",
              "Value": "sp",
            },{
              "Type": "group",
              "Value": "group1",
            },{
              "Type": "group",
              "Value": "group2",
            }]
          }
        ]
      IDENTITY_RESOURCES_INLINE: |
          [
            {
              "Name": "tenant",
              "ClaimTypes": ["tenantId"]
            }, {
              "Name": "roles",
              "ClaimTypes": ["role"]
            }, {
              "Name": "groups",
              "ClaimTypes": ["group"]
            },
          ]`

I'm not sure if such customization should be added to readme example

### CODE REVIEW

**Comment by vitalyal** [comment]
Hello @AleF83, 
Could you please provide feedback on this PR and maybe tell what needs to be additionally done to get it merged

**Comment by AleF83** [comment]
Hey, Vitaly.
Thank you for your PR.
Sorry for late response.
It looks great!
Just one small remark.
After you'll fix it I'll merge it to `dev` branch and when it will pass CI I'll merge it to `master` branch and push image to DockerHub

File: src/utils/ClaimJsonConverter.cs [hunk 1]
```diff
@@ -0,0 +1,32 @@
+using System;
+using System.Security.Claims;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+
+namespace OpenIdConnectServer.Utils
+{
+    public class ClaimJsonConverter : JsonConverter
```
[the remark this answers is not in the document]
Comment (vitalyal) [reply src/utils/ClaimJsonConverter.cs]: thank you, missed that, it is  fixed now

**Comment by AleF83** [comment]
@vitalyal I've pushed new image to DockerHub.
Thank you for your contribution!

**Comment by vitalyal** [comment]
@AleF83 thank you for publishing and for the project itself! :)

**Pull request merged**